src/__tests__/app/create-template/page.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@testing-library/react';
+import CreateTemplate from '@/src/app/create-template/page';
```

Comment:
we've got a ticket planned to stop using @/src - until we do that, should we be using relative paths at least for new code?

Reply:
let's leave it as is since it'll be an easier find and replace.